### DISCUSSION TITLE
Where is 0.7.0?

### DISCUSSION THREAD

**Discussion by TechStudent10** [geode-sdk/sdk#98] [Help] [answered]
I cannot find version 0.7.0 of Geode. I need it for a mod I'm building.

**Comment by HJfod** [comment]
0.7.0 has not been released yet (and there is a chance it never will, as we have made so many changes to the framework the next release might just be v1.0.0). Target v0.6.1 for now

**Reply by TechStudent10** [reply]
I need `acbyass` and that needs 0.7.0.

**Comment by HJfod** [comment, marked as answer]
Oh, that sounds like an error with the mod. It should not target 0.7.0.

**Reply by TechStudent10** [reply]
Ah okay. Let me go check.

**Reply by TechStudent10** [reply]
https://github.com/camila314/geode-mods/blob/main/acbypass/mod.json

![image](https://user-images.githubusercontent.com/76978184/207135298-699a70f1-4ee1-4126-8390-7f79668765ad.png)

**Reply by TechStudent10** [reply]
Targets Geode `0.7.0` weird.

**Reply by TechStudent10** [reply]
Have a solution?
